recipes/rocksdb/all/test_package/conanfile.py

```diff
@@ -0,0 +1,23 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.definitions["ROCKSDB_DLL"] = self.options["rocksdb"].shared
+
+        cmake.configure()
+        cmake.build()
+
```

Comment:
```suggestion

    def config_options(self):
        minimal_cpp_standard = "11"
        if self.settings.compiler.cppstd:
            tools.check_min_cppstd(self, minimal_cpp_standard)

```

Would it make sense, as you did in the conanfile.py ?